### PR TITLE
Add Subscription affinity configuration documentation

### PR DESCRIPTION
**Pull request by perdasilva**
Signed-off-by: perdasilva <perdasilva@redhat.com>

### CODE REVIEW

File: content/en/docs/advanced-tasks/overriding-operator-pod-affinity-configuration.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  config:
+  	affinity:
+  		nodeAffinity:
+	      requiredDuringSchedulingIgnoredDuringExecution:
+	        nodeSelectorTerms:
+	        - matchExpressions:
+	          - key: kubernetes.io/arch
+	            operator: In
+	            values:
+	            - amd64
```
Comment (timflannagan) [inline content/en/docs/advanced-tasks/overriding-operator-pod-affinity-configuration.md:57]: This is rendering weird when looking at the diff. Whitespace issues?

File: content/en/docs/advanced-tasks/overriding-operator-pod-affinity-configuration.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+metadata:
+  name: my-package
+  namespace: my-namespace
+spec:
+  name: my-package
+  source: my-operators
+  sourceNamespace: operator-registries
+  config:
+  	affinity:
+  		nodeAffinity:
```
Comment (timflannagan) [inline content/en/docs/advanced-tasks/overriding-operator-pod-affinity-configuration.md:50]: This indentation (and a couple of other examples) looks off compared to the rest of the YAML document.

**Review by timflannagan — COMMENTED** [review]
LGTM once we fix the whitespace and indentation issues.

File: content/en/docs/advanced-tasks/overriding-operator-pod-affinity-configuration.md (hunk 1 above)
Comment (perdasilva) [reply content/en/docs/advanced-tasks/overriding-operator-pod-affinity-configuration.md]: fixed

File: content/en/docs/advanced-tasks/overriding-operator-pod-affinity-configuration.md (hunk 2 above)
Comment (perdasilva) [reply content/en/docs/advanced-tasks/overriding-operator-pod-affinity-configuration.md]: fixed

**Review by anik120 — APPROVED** [review]

**Review by grokspawn — APPROVED** [review]

**Pull request merged**